fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CompletedSnapshotStoreManager.java

```diff
@@ -96,6 +97,14 @@ public CompletedSnapshotStore getOrCreateCompletedSnapshotStore(TableBucket tabl
                 });
     }
 
+    public void onRemoveCompletedSnapshotStoreByTableBuckets(Set<TableBucket> tableBuckets) {
```

Comment:
nit:
```suggestion
    public void removeCompletedSnapshotStoreByTableBuckets(Set<TableBucket> tableBuckets) {
```
to make the naming style align with `createCompletedSnapshotStore`